csrc/logical_domain_map.h

```diff
@@ -466,6 +466,10 @@ class ComputeAtLogicalDomainMapBuilder : private BackwardVisitor {
     mapPointwiseLikeOp(op);
   }
 
+  void handle(IndexPutAccumulateOp* op) override {
+    mapPointwiseLikeOp(op);
+  }
+
```

Comment:
Is this required if you are rejecting it in schedulers other than `ExprEval`?

Reply:
That's a good call. I don't think I need this until we enable codegen support.
I'll defer this change then.